libraries/AP_HAL_ChibiOS/RCOutput.cpp

```diff
@@ -1693,7 +1695,8 @@ void RCOutput::dshot_send(pwm_group &group, uint64_t time_out_us)
 #if HAL_SERIALLED_ENABLED
 bool RCOutput::serial_led_send(pwm_group &group)
 {
-    if (!group.serial_led_pending || !is_led_protocol(group.current_mode)) {
+    if (!group.serial_led_pending || !is_led_protocol(group.current_mode)
+        || AP_HAL::micros64() - group.last_dmar_send_us < (group.dshot_pulse_time_us + 50)) {
```

Comment:
shouldn't we return false in this new "to fast" case so it will try again next time? Currently I think this means we might loose data.

Reply:
good point

Reply:
Fixed